### PR TITLE
Graph generators and SQLite Exposed coverage with tests and some fixes to Graph saving and generating

### PR DESCRIPTION
**Pull request by MuhammetSultanov**
1) Decided to remove support for Char in graph saving and generating for reasons below:
 - When saving string "A" in SQLite, it confuses it's type with char 'A'.
 - It is more efficient to save char as a string because it will take approximately same amount storage, but it will be easier to deserialize saved data from SQLite DB.
2) Covered with tests graph generators (Random with fixed weight 1, Random with random weight, Flower snark, Star directed, Star undirected).
3) Covered with tests Save, Get and Delete functions of graph from SQLite DB.
4) Covered with tests Save and Update state of theme functions in SQLite DB.
5) Added tests to check that generators create and connect nodes only from the graph itself, not from outer sources.

### CODE REVIEW

File: src/test/kotlin/dataTest/GraphGeneratorsTest.kt [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+                        graph.vertices.values.flatten().forEach { (_, weight) ->
+                            assertTrue { weight <= maxWeight.toFloat() && weight > 0f }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (AlexandrKudrya) [inline src/test/kotlin/dataTest/GraphGeneratorsTest.kt:161]: Добавь тесты инвариантов

File: src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                addVertice(3, 4)
+                addVertice(3, 5)
+                addVertice(5, 8)
+                addVertice(5, 6)
+                addVertice(6, 9)
+                addVertice(7, 8)
+                addVertice(7, 10)
+                addVertice(8, 9)
+                addVertice(9, 10)
+
```
Comment (AlexandrKudrya) [inline src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt:31]: Много для unit, не очень понятно зачем

File: src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                addVertice("C", "D")
+                addVertice("C", "E")
+                addVertice("E", "H")
+                addVertice("E", "F")
+                addVertice("F", "I")
+                addVertice("G", "H")
+                addVertice("G", "J")
+                addVertice("H", "I")
+                addVertice("I", "J")
+            }
```
Comment (AlexandrKudrya) [inline src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt:41]: Многа

File: src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+                addVertice(3f, 4f)
+                addVertice(3f, 5f)
+                addVertice(5f, 8f)
+                addVertice(5f, 6f)
+                addVertice(6f, 9f)
+                addVertice(7f, 8f)
+                addVertice(7f, 10f)
+                addVertice(8f, 9f)
+                addVertice(9f, 10f)
+            }
```
Comment (AlexandrKudrya) [inline src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt:49]: Многа

File: src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    @Nested
+    inner class SerializeAndDeserializeTests {
+
+        @Test
+        fun `test serializeGraph function with Int`() {
+            val serializedGraph = serializeGraph(intGraph)
+            val expectedSerializedGraph =
+                "1:(2, 1.0);(4, 1.0)|2:(3, 1.0);(5, 1.0);(7, 1.0)|3:(4, 1.0);(5, 1.0)|4:|5:(8, 1.0);(6, 1.0)|6:(9, 1.0)|7:(8, 1.0);(10, 1.0)|8:(9, 1.0)|9:(10, 1.0)|10::10"
```
Comment (AlexandrKudrya) [inline src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt]: Заменить бы на генератор, а не чтобы просто строка в тестах лежала

Comment (AlexandrKudrya) [reply src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt]: Ну типо если менять, то плохо

File: src/test/kotlin/dataTest/GraphGeneratorsTest.kt (hunk 1 above)
Comment (MuhammetSultanov) [reply src/test/kotlin/dataTest/GraphGeneratorsTest.kt:161]: Добавил

File: src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt (hunk 2 above)
Comment (MuhammetSultanov) [reply src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt:31]: Уменьшил

File: src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt (hunk 5 above)
Comment (MuhammetSultanov) [reply src/test/kotlin/dataTest/SQLiteExposedGraphTest.kt]: Уменьшил текст чтобы было легче читать.

**Pull request merged**